src/libraries/System.IO.Compression/src/System/IO/Compression/DeflateZLib/ZLibException.cs

```diff
@@ -53,20 +53,16 @@ public ZLibException(string? message, Exception? innerException) : base(message,
         /// </summary>
         /// <param name="info">The SerializationInfo that holds the serialized object data about the exception being thrown.</param>
         /// <param name="context">The StreamingContext that contains contextual information about the source or destination.</param>
-#if NET8_0_OR_GREATER
```

Comment:
Since the `System.IO.Compression` nuget package is no longer being updated, I assume the code only applies to the latest CLR. Should the entire method be removed?

Reply:
Does this mean that the project no longer targets older TFMs?

Reply:
> Does this mean that the project no longer targets older TFMs?

Yes.

https://github.com/dotnet/runtime/blob/386669e7bb97ce977c4a0629eed8d6955e72298c/src/libraries/System.IO.Compression/src/System.IO.Compression.csproj#L4

Reply:
> Does this mean that the project no longer targets older TFMs?

Correct.

https://github.com/dotnet/runtime/blob/386669e7bb97ce977c4a0629eed8d6955e72298c/src/libraries/System.IO.Compression/src/System.IO.Compression.csproj#L4

Reply:
SGTM